app/views/components/gds/gds_main_template.scala.html

```diff
@@ -38,7 +38,8 @@
 @(
   title: Title,
   backButton: Option[BackButton] = None,
-  useCustomContentWidth: Boolean = false
+  useCustomContentWidth: Boolean = false,
+  showSignOutLink: Boolean = true
```

Comment:
Could we infer this variable (showSignOutLink) based on whether the user is logged in or not (like the useTimeoutDialog method below)?

Reply:
Good point. Didn't think about it this way.

Reply:
I went down this path with timeout dialog.  Had a default param that all pages would have the dialog unless they said they didn't need it because they were used when the user was not logged in.